### ISSUE TITLE
Add 'sole owner' badge on list of projects

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#3127]
We've had a lot of positive feedback on the bus factor feature on the account settings page.

It would be nice to also expose this information on `/manage/projects/` so that users don't have to hunt for it :)

**Comment by nlhkabu** [comment]
Something like this using the `<span class="badge>` component would be ideal:

![screenshot from 2018-03-03 15-35-22](https://user-images.githubusercontent.com/3323703/36936109-834c78ca-1ef8-11e8-81dc-e5fbbb5fcddc.png)

**Comment by NotAFile** [comment]
How would you recommend fetching the status for this?

On the account settings page, this is done with a separate query, but I'd be reluctant to add another db query to this frequently used page just for the one badge.

**Comment by NotAFile** [comment]
@di yes, but that was what I was trying to avoid. It's basically two queries on the exact same thing. In django, I'd use the annotate functionality, but this doesn't seem to be available in SQAlchemy. I was wondering how you'd recommend doing this for this project. Two seperate queries and a membership check seems like the worst way to do it, but I might be missing something here.

**Comment by dstufft** [comment]
I would be very surprised if what you want can't be done with SQLAlchemy. Possibly using the SubQuery or Aliasing support.

**Comment by gabrielgradinaru** [comment]
Can I try this one?

**Comment by nlhkabu** [comment]
Yes, sounds great :)

**Comment by rupali-saboo** [comment]
I'd like to try this for the PyPA sprint
